src/components/bankaccounts/BankAccountsEditForm.tsx

```diff
@@ -33,7 +35,7 @@ export default function BankAccountsEditForm({ id }: Props) {
     accountHolderType: data?.accountHolderType,
     bankName: data?.bankName,
     bankIdCode: data?.bankIdCode,
-    fingerprint: data?.fingerprint,
+    // fingerprint: data?.fingerprint,
```

Comment:
@AndreyGoranov Why do we remove the `fingerprint` here? Looks like this is a temporary local solution to some kind of problem.

Reply:
The ticket says: "fingerprint to be hidden - not needed for now", so should I delete it?